src/renderer/contexts/SettingsContext.tsx

```diff
@@ -60,7 +60,7 @@ export const SettingsProvider = memo(({ children }: React.PropsWithChildren<unkn
     const useSystemPythonLocation = useMemoArray(
         useLocalStorage<string | null>('system-python-location', null)
     );
-    const useCheckUpdOnStrtUp = useMemoArray(useLocalStorage('check-upd-on-strtup', true));
+    const useCheckUpdOnStrtUp = useMemoArray(useLocalStorage('check-upd-on-strtup-2', true));
```

Comment:
Do we even need a setting for update checks anymore? The update "notification" is very non-intrusive now, why shouldn't we just always check for updates?

Reply:
I agree, but I guarantee you somebody will complain if it's removed completely

Reply:
Alright. I can see that.